2023/09.md

```diff
@@ -10,6 +10,21 @@
 - **Location**: TOKYO, JAPAN
 - **Attendee information**: https://github.com/tc39/Reflector/issues/499
 
+```mermaid
+---
+displayMode: compact
+---
+gantt
+    title 98th meeting
+    dateFormat  YYYY MM DD HH mm Z
+    tickInterval 3d
```

Comment:
```suggestion
```

`tickInterval` syntax is incorrect (which is why it is displaying default of ticks at `1day`.  I don't think you would want 3day ticks, so just remove.